src/panel/direction/RoutesList.jsx

```diff
@@ -63,4 +69,19 @@ const Placeholder = () => {
   );
 };
 
+const moveRouteToTop = (routes, id) => {
```

Comment:
This seems really complicated for just that, and find/findIndex still don't exist on IE 11 :P
What about using sort?
```js
return routes
    .slice() // clone the array as sort operates on-place
    .sort((a, b) => a.id === id ? -1 : (b.id === id ? : 1 : 0));
```